src/condor_tests/ornithology/plugin.py

```diff
@@ -9,17 +9,52 @@
 
 @pytest.fixture(scope="session")
 def path_to_sleep():
+    """
+    This fixture returns the full path pointing to a Python sleep
+    executable.
+
+    Returns
+    -------
+     :class:`str`
+        Full path to Python sleep executable.
+    """
     return SCRIPTS["sleep"]
 
 
 @pytest.fixture(scope="session")
 def path_to_null_plugin():
+    """
+    This fixture returns the full path pointing to the Python NULL file
+    transfer plugin.
+
+    .. note::
+
+        This plugin is added to the default File Transfer Plugin list.
```

Comment:
```suggestion
        This plugin is in Ornithology's default File Transfer Plugin list.
```
To be clear that calling the fixture doesn't change the condor config somehow.